fud2/src/lib.rs

```diff
@@ -282,8 +282,7 @@ pub fn build_driver(bld: &mut DriverBuilder) {
         e.build_cmd(&[only_refs_calyx], "external-to-ref", &[input], &[])?;
 
         // Get YXI to generate JSON for testbench generation
-        e.build_cmd(&[memories_json], "calyx", &[only_externals_calyx], &[])?;
-        e.arg("backend", "yxi")?;
+        e.build_cmd(&[memories_json], "yxi", &[only_externals_calyx], &[])?;
```

Comment:
Tried to address the fud2 changes for firrtl as well @ayakayorihiro. Lmk if this looks ok?

Reply:
This looks great, thanks!!